DuckDuckGo/TabViewController.swift

```diff
@@ -1442,14 +1442,18 @@ extension TabViewController: EmailManagerRequestDelegate {
 
     // swiftlint:disable function_parameter_count
     func emailManager(_ emailManager: EmailManager,
-                      didRequestAliasWithURL url: URL,
+                      requested url: URL,
                       method: String,
                       headers: [String: String],
+                      parameters: [String: String]?,
+                      httpBody: Data?,
                       timeoutInterval: TimeInterval,
                       completion: @escaping (Data?, Error?) -> Void) {
         APIRequest.request(url: url,
                            method: APIRequest.HTTPMethod(rawValue: method) ?? .post,
+                           parameters: parameters,
                            headers: headers,
+                           httpBody: httpBody,
                            timeoutInterval: timeoutInterval) { response, error in
```

Comment:
FWIW I talked to @brindy earlier about how the app needs to use delegation from BSK just to make a network request. I'll write up a task to look at moving the API request class into BSK so we can get away from this, it needs to be done anywhere an EmailManager is used.